react-native-libraries.json

```diff
@@ -7191,8 +7191,10 @@
     "githubUrl": "https://github.com/Giphy/giphy-react-native-sdk",
     "npmPkg": "@giphy/react-native-sdk",
     "examples": ["https://github.com/Giphy/giphy-react-native-sdk/tree/main/example"],
+    "images": ["https://raw.githubusercontent.com/Giphy/giphy-react-native-sdk/main/docs/assets/example_app.gif"],
     "ios": true,
-    "android": true
+    "android": true,
+    "newArchitecture": true
```

Comment:
Hello @pshoniuk, thank you for updating the directory entry! 👍 

```suggestion
    "android": true
```

There is no need to manually add new architecture flag, since library is already marked correctly by the autodetection methods:

<img width="563" alt="Screenshot 2023-08-17 094130" src="https://github.com/react-native-community/directory/assets/719641/f6b3cfba-0c44-472b-94cc-faff87115dff">